packages/base/card-api.gts

```diff
@@ -2300,6 +2302,56 @@ export class MaybeBase64Field extends StringField {
   static atom = MaybeBase64Field.embedded;
 }
 
+export class TextAreaField extends StringField {
+  static displayName = 'TextArea';
+  static icon = TextAreaIcon;
+  static edit = class Edit extends Component<typeof this> {
+    <template>
+      <BoxelInput
+        class='boxel-text-area'
+        @value={{@model}}
+        @onInput={{@set}}
+        @type='textarea'
+        @disabled={{not @canEdit}}
+      />
+    </template>
+  };
+}
+
+export class CSSField extends TextAreaField {
+  static displayName = 'CSS Field';
+  static embedded = class Embedded extends Component<typeof this> {
+    <template>
+      <pre class='css-field'>
+        {{if
+          @model
+          @model
+          '/* No CSS defined */'
+        }}
+      </pre>
+      <style scoped>
+        .css-field {
+          margin-block: 0;
+          padding: var(--boxel-sp-xxs);
+          background-color: var(--muted, var(--boxel-100));
+          border-radius: var(--radius, var(--boxel-border-radius));
+          color: var(--muted-foreground, var(--boxel-700));
+          font-family: var(
+            --font-mono,
+            var(--boxel-monospace-font-family, monospace)
+          );
+          white-space: pre-wrap;
+        }
+      </style>
+    </template>
+  };
+}
+
+export class CardInfoField extends FieldDef {
+  static displayName = 'Card Info';
+  @field theme = linksTo(() => Theme);
+}
```

Comment:
card-api is pretty big as it is. i'm always looking for opportunities to reduce it in size. can we move these out into their own modules?

Reply:
the issue is I actually had to move more things in because somehow we can't import them. it's the field-defs and card-defs